packages/builder/ts/setup.ts

```diff
@@ -0,0 +1,4 @@
+// @ts-ignore
```

Comment:
Same as above.

Reply:
We could probably move this out of the `builder` now all together, as it's only api related. It should probably live it qa-core itself